calculate_imputation_factors_wrangler.py

```diff
@@ -91,8 +97,15 @@ def lambda_handler(event, context):
         if not json_response['success']:
             raise exception_classes.MethodFailure(json_response['error'])
 
+        output_df = pd.read_json(json_response['data'])
+        distinct_values.append(period_column)
```

Comment:
Related to the above comment. Do we even need period now as this is a dataframe of factors, region and strata?